content/1.1.1.1/_index.md

```diff
@@ -13,7 +13,7 @@ Speed up your online experience with Cloudflare's public DNS resolver.
 
 {{<plan type="all">}}
 
-1.1.1.1 is Cloudflare’s public DNS resolver. It offers a fast and private way to browse the Internet. [DNS resolvers](https://www.cloudflare.com/learning/dns/what-is-dns/) translate domains like `cloudflare.com` into the IP addresses necessary to reach the website (like `104.16.123.96`).
+1.1.1.1 is Cloudflare’s “public” DNS resolver. It offers a fast and private way to browse the Internet. [DNS resolvers](https://www.cloudflare.com/learning/dns/what-is-dns/) translate domains like `cloudflare.com` into the IP addresses necessary to reach the website (like `104.16.123.96`).
```

Comment:
**[rdjson]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>**Warning**: Use standard single quotes or double quotes only. Do not use left or right quotes. For more details, refer to [Quotation marks](https://developers.cloudflare.com/style-guide/grammar/punctuation-marks-and-symbols/quotation-marks/).
```suggestion
1.1.1.1 is Cloudflare's "public" DNS resolver. It offers a fast and private way to browse the Internet. [DNS resolvers](https://www.cloudflare.com/learning/dns/what-is-dns/) translate domains like `cloudflare.com` into the IP addresses necessary to reach the website (like `104.16.123.96`).3.96`).
```